packages/cosmo/package.py

```diff
@@ -53,6 +55,14 @@ class Cosmo(MakefilePackage):
     variant('claw', default=False, description='Build with claw-compiler')
     variant('slave', default='tsa', description='Build on slave tsa or daint', multi=False)
     variant('eccodes', default=False, description='Build with eccodes instead of grib-api')
+    variant('production', default=False, description='Force all variants to be the ones used in production')
+    
+    conflicts('+production', when='~cppdycore')
+    conflicts('+production', when='+serialize')
+    conflicts('+production', when='+debug')
+    conflicts('+production', when='~claw')
+    conflicts('+production', when='~parallel')
+    conflicts('+production', when='cosmo_target=cpu')
```

Comment:
actually I wanted to say
`when='cosmo_target!=gpu'` but that syntax does not work.
Do you know @elsagermann how to specify the not?